src/Resolver/DependencyResolverInterface.php

```diff
@@ -32,24 +33,23 @@ public function setContainer(ContainerInterface $container);
     /**
      * Resolve a type prefernece
      *
-     * @param string $type The type/class name of the dependency to resolve the
-     *     preference for
-     * @param string $context The typename of the instance that is created or
+     * @param string      $type The type/class name of the dependency to resolve the
+     *          preference for
```

Comment:
This looks also a bit off. Was this also applied by the auto-fixer?

Reply:
Those multiline DocBlock got "fixed" automatically.

Reply:
I see, thanks.

@boesing should it look like this. Maybe this is some edge case the auto-fixer ran into? Shouldn't this be just 4 spaces of indentation for each new line?

Reply:
> @boesing should it look like this. Maybe this is some edge case the auto-fixer ran into? Shouldn't this be just 4 spaces of indentation for each new line?

TBH - I dont know. I'd say, if the phpcbf fixes the code like this and it wasnt applied manually, we should keep this as is and either wait until there is a bugfix upstream. 🤷‍♂️ 
Feel free to report that.

Reply:
Agree